### DISCUSSION TITLE
implement graphrag

### DISCUSSION THREAD

**Discussion by arjunkrishna** [open-webui/open-webui#3647] [Ideas]
I came across this video today about GraphRAG. looks interesting.  https://www.youtube.com/watch?v=f7Puiilv5Tw

**Comment by pbpippen** [comment]
agree this would be good https://github.com/microsoft/graphrag

**Comment by arjunkrishna** [comment]
ah... the embeddings are still being worked upon in ollama. https://github.com/ollama/ollama/pull/5285

https://github.com/microsoft/graphrag/issues/339#issuecomment-2210367643

**Reply by pbpippen** [reply]
someone got it on ollama and lmstudio
https://x.com/mervinpraison/status/1809480607237275754

**Comment by edisonzf2020** [comment]
+1

**Comment by ma3oun** [comment]
The embedding api is up and running: https://github.com/ollama/ollama/blob/main/docs/api.md#generate-embeddings
